types/error/errInvalidPermissions.go

```diff
@@ -0,0 +1,13 @@
+package types
```

Comment:
Please rename filename to `invalid_permissions.go`, just to follow filename conventions.